source/stylesheets/header.scss

```diff
@@ -1,49 +1,50 @@
 details > summary {
-	padding: 1.05em;
-	color: white;
-	font-size: 1.8em;
+  padding: 1.05em;
+  color: white;
+  font-size: 1.8em;
 }
 
 details {
-	background: #ed3e44;
-	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
-}
+  background: #ed3e44;
+  box-shadow: $base-box-shadow;  
+  }
 
 details[open] {
-	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
+  box-shadow: $base-box-shadow;  
 
 }
 
 nav {
-	color: white;
-	position: relative;
+  color: $white;
+  position: relative;
 }
 
 details ul {
-	padding: 0;
-	margin: 0;
+  padding: 0;
+  margin: 0;
 }
 
 li {
- display: block;
+  display: block;
 }
 
 details li {
-	background-color: #4C95C7;
-	border-bottom: 1px solid #65FFFF;
+  background-color: #4C95C7;
+  border-bottom: 1px solid #65FFFF;
 }
 
 details li:hover {
-	background-color: #657EA8;
-}
+  background-color: #657EA8;
+  }
+
 details a {
-	display: block;
-	padding: 1.25em;
-	color: white;
-	text-decoration: none; 
+  display: block;
+  padding: 1.25em;
+  color: white;
+  text-decoration: none; 
 }
 
-summary:hover {
-	cursor: pointer;
-	background-color: #F0AA41;
-}
+  summary:hover {
+  cursor: pointer;
+  background-color: #F0AA41;
```

Comment:
I'd walk through this file again to make sure you're getting all the colors into variables, we're missing a few here. That way we're less likely to have different shades of yellow throughout the project, for example, and we can also change them all at once.

Likewise, if you want to derive this shade of yellow from the normal state, Sass provides some [really cool color functions](https://robots.thoughtbot.com/controlling-color-with-sass-color-functions). So this could become `color: darken($yellow, 5%)` for example.

Reply:
I don't want to apply variables across files in this PR if I'm going to upload other styling that might override those files. Not sure if certain files will override those changes in the other PRs for button styling, different header styling/spacing, etc.

I think it might be best to push this PR, as well as the other PRs for button and navigation, into master, then from master apply variables across the project with the most updated elements and note it in the next PR.

Reply:
The concern is that you have other pull requests touching these parts too?

If that's the case, I think it's worth doing so we can walk through resolving conflicts together. I think it'd be worthwhile for the experience.